packages/trader/src/Stores/Modules/Contract/Helpers/chart-notifications.js

```diff
@@ -9,5 +9,6 @@ export const switch_to_tick_chart = {
             i18n_default_text='Please change the chart duration to tick for a better trading experience.'
         />
     ),
-    type: 'info',
+    type         : 'info',
+    is_persistent: true,
```

Comment:
Why does this notification have to be persistent? Persistent stays across platform changes too (read: Trader to MT5).

Reply:
Ah, nice catch. I forgot to remove this, realized the same thing when I tested, hence it's not in the Changelog.